ldms/doc/mdlinkfilter.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
```

Comment:
@narategithub can you update this to "#! /usr/bin/env python3" so that mdlinkfilter will work on both rhel7 and rhel8?
may be  a few py3 syntax changes needed to do this.